File: 0392-is-subsequence/0392-is-subsequence.c
bool isSubsequence(char * s, char * t){
    int i = 0, j = 0;
    int m = strlen(s), n = strlen(t);
    
    while(i < m && j < n){
        if(s[i] == t[j]){
            i++;
        }
        j++;
    }
    return i == m;
}

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        i, j, m, n = 0, 0, len(s), len(t)
        while i < m and j < n:
            if s[i] == t[j]:
                i += 1
            j += 1
        return i == m
